jlm/rvsdg/graph.cpp

```diff
@@ -22,26 +22,11 @@ GraphImport::GraphImport(
       Name_(std::move(name))
 {}
 
-/* expport */
-
-expport::~expport()
+GraphExport::GraphExport(rvsdg::output & origin, std::string name)
+    : result(origin.region()->graph()->root(), &origin, nullptr, origin.Type()),
+      Name_(std::move(name))
```

Comment:
@haved The constructor explicitly takes the root region and hands it over to the origin. I do not see where you want to put the assert?

Reply:
Ah, sorry I didn't catch that. So if you try to make a `GraphExport` for an output that is not in the root region, a different assert will complain that the output and result belong to different regions.

It all looks good, go ahead and merge, @phate

Reply:
@haved Yes, exactly. If the origin's region is not the root region, then an exception should be thrown [here](https://github.com/phate/jlm/blob/d77c368689d5cce5ac8b43e28ded9dca1b51bd21/jlm/rvsdg/node.cpp#L33C1-L34C55).